JS simple function/index.js

```diff
@@ -32,3 +32,23 @@ export function func(s, a, b) {
         return -1;
     }
 }
+
+/**
+ * refactored function
+ */
+
+ export function func(s, a, b) {
+
+    let result = -1
+
+    switch (s.length) {
+        case 0 || 1: return result
```

Comment:
Это условие записано некорректно. Дело в том, что оператор `||` тут всегда вернет `1` (потому что `0` falsy, см. [тут](https://learn.javascript.ru/logical-operators#ili-nahodit-pervoe-istinnoeznachenie)) Например:
```
function zeroOrOne(p) {
    switch (p) {
        case 0 || 1: return true;
        default: return false;
    }
}
zeroOrOne(0); // false !!!
zeroOrOne(1); // true
zeroOrOne(2); // false
```
Если нужна проверка на 0 или 1, стоило написать
```suggestion
        case 0:
        case 1: return result;
```

Reply:
совершенно верно, спасибо за объяснение.